### PR TITLE
fix highs for bazel 8

### PR DESCRIPTION
**Pull request by lperron**
Fix missing files in glob

bump dependencies 

add patch file vs fresh github clone

@bazel-io skip_check unstable_url

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (highs) have been updated in this PR. Please review the changes.

File: modules/highs/1.8.1.bcr.1/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  bazel:
+  - 7.x
```
Comment (hofbi) [inline modules/highs/1.8.1.bcr.1/presubmit.yml:8]:
```suggestion
  - 8.x
  - 7.x
```

**Comment by hofbi** [comment]
> Hi Markus, I should not have committed your patch, you need to sign the CLA

I have signed the CLA, looks like the proposal just too the wrong mail. Feel free to squash locally to get rid of my contribution

**Comment by hofbi** [comment]
```
git checkout highs-bazel8  # make sure you have all local changes available
git reset HEAD~2  # this will undo the last two commits
# commit again
git push --force your_remote your_branch
```

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**